packages/core/src/payment-integration/create-payment-integration-selectors.spec.ts

```diff
@@ -14,9 +14,11 @@ describe('createPaymentIntegrationSelectors', () => {
         beforeEach(() => {
             internalSelectors = createInternalCheckoutSelectors(getCheckoutStoreStateWithOrder());
             subject = createPaymentIntegrationSelectors(internalSelectors);
+            console.log('This runs', subject.getBillingAddress);
         });
 
-        it('returns copy of billing address', () => {
+        it.only('returns copy of billing address', () => {
+            console.log('This runs second', subject);
```

Comment:
Learned a lot from both of you, @animesh1987 and @davidchin , on this PR. 🙇🏻
This looks like some midway work, is it intended to be?

Reply:
No, I don't think it is intentional. Good catch! I'll update now.

Reply:
Nice catch @bc-peng